src/smc-webapp/projects.cjsx

```diff
@@ -525,7 +525,7 @@ class ProjectsStore extends Store
             return 0
         others = []
         for i in v
-            if not i.deleted and (show_hidden or not i.get('users').get(account_id).get('hide'))
+            if (not (i.get('deleted') == true )) and (show_hidden or not i.get('users').get(account_id).get('hide'))
```

Comment:
Any reason for the preferred syntax? I wasn't sure what that would return since projects which aren't deleted simply don't have a 'deleted' key/property in their map.  If 'deleted' isn't true this is avoided.